src/test/java/com/exasol/adapter/dialects/postgresql/PostgresVirtualSchemaIntegrationTestSetup.java

```diff
@@ -86,10 +95,12 @@ public class PostgresVirtualSchemaIntegrationTestSetup implements Closeable {
     private static void uploadDriverToBucket(final Bucket bucket)
             throws InterruptedException, TimeoutException, BucketAccessException {
         try {
-            bucket.uploadFile(JDBC_DRIVER_PATH, JDBC_DRIVER_NAME);
+            bucket.uploadStringContent(JDBC_DRIVER_CONFIGURATION_FILE_CONTENT,
+                    "drivers/jdbc/" + JDBC_DRIVER_CONFIGURATION_FILE_NAME);
+            bucket.uploadFile(JDBC_DRIVER_PATH, "drivers/jdbc/" + JDBC_DRIVER_NAME);
```

Comment:
Please replace this with the [driver manager](https://github.com/exasol/exasol-testcontainers/blob/main/doc/user_guide/user_guide.md#installing-drivers-for-external-data-sources-and-sinks) from exasol-testcontainers.

Reply:
done!